### PR TITLE
Newsweek: user-interface

### PR DESCRIPTION
**Pull request by ampaire**
The [Bootstrap](https://v4-alpha.getbootstrap.com/) framework is used to develop the [Neewsweek](https://www.newsweek.com/) clone

*[Original project](https://www.newsweek.com/)
*[My clone](https://raw.githack.com/ampaire/Newsweek-page/ft-homepage/index.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1593 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (bolah2009) [inline index.html]: Let's add bootstrap's responsive viewport meta tag.
```html
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
See [here](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag)

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,811 @@
...
+  width: 100%;
+}
+
+.item-3,
+.item-2,
+.item-1 {
+  position: relative;
+  font: bold 14px/14px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+  display: inline-block;
```
Comment (bolah2009) [inline css/styles.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class name can be used instead. See [Display](https://getbootstrap.com/docs/4.1/utilities/display/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,811 @@
...
+}
+
+img {
+  width: 100%;
+}
+
+.item-3,
+.item-2,
+.item-1 {
+  position: relative;
```
Comment (bolah2009) [inline css/styles.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class name can be used instead. See [Position](https://getbootstrap.com/docs/4.1/utilities/position/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome Job @ampaire  :clap: 

###  The [clone](https://rawcdn.githack.com/ampaire/Newsweek-page/53962331ed1c18fd90a1970d3dddf9b302d4e846/index.html) looks good to me and very responsive.. :smile: 

A few required changes :relaxed: 
- [REQUIRED] Let's use more of bootstrap as pointed out in the comments below (and in all other occurrences wherever possible) as **Using Bootstrap** is the main objective of this project.
  > Remember to use Bootstrap CSS classes wherever it is possible - this is the main point of this project.
- [REQUIRED] Make navbar responsive for smaller screens
- Let's make this Pull Request look more professional.
  - [NEXT PROJECT]: Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) and Check [this awesome article](https://chris.beams.io/posts/git-commit/)
  - [REQUIRED] Use meaningful names for  Pull Requests.
  - [REQUIRED] Add a short summary of changes in Pull Request description.

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — COMMENTED** [review]
# Let's make this Pull Request look more professional.
  - [NEXT PROJECT]: Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) and Check [this awesome article](https://chris.beams.io/posts/git-commit/)
  - [REQUIRED] Use meaningful names for  Pull Requests.
  - [REQUIRED] Add a short summary of changes in Pull Request description.

# Rename PR
  rename PR
-----
# Update PR description
  update-PR-description-html

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @kitpao 

### Nice work so far, however please make the following changes:

- Make the navbar responsive for mobile screens.
- implement all the changes requested in the previous code review.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by Forison — COMMENTED** [review]
# pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
# Hi @ampaire 
the project looks great but here are a few things I would like you to address 🙂🙂
- Navbar is not as responsive as it should🙂🙂
> The [newsweek](newsweek.com) has at least three breakpoints for the navigation. It changes at XS, SM, MD 
> Use bootstrap classes as much as possible. Bootstrap has got classes for navbar implementation
refer [to](https://getbootstrap.com/docs/4.3/components/navbar/)🙂🙂
> bootstrap4 was built mainly on Flex display properties you must use those classes instead of using classes like position relative. refer [to](https://getbootstrap.com/docs/4.3/components/navbar/)
- Take a  careful look  at the way each section gets ordered at the different breakpoint
> top story first, featured story second, followed by the opinion🙂🙂
## Kindly do the changes suggested and submit for review

##### Guys, please fix the errors I have pointed out and submit your project for another review
feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.

Good luck next time🙂🙂

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Awesome job :+1: @ampaire 
You have met the most required features for this project. That is good.
But before we can merge, please make one change below;
- [ ] remove comments from your code especially from ```style.css```

Please make those changes and re-submit for another review.

**Comment by ampaire** [comment]
@whiz25  I am not working on the project you are telling me. I am working on the Newsweek page.

**Comment by whiz25** [comment]
@ampaire sorry about that. It was a mistake. Please look at the updated comment.

**Review by uche-inyama — APPROVED** [review]
Hi @ampaire, good job so far,

You navigation bar implementation is not properly done: 
- [ ] it is not taking the full width of the screen size; kindly fix this.

- [ ] The newsweek logo, moves to the left of the navigation bar, from screen sizes less than or eqaul to large screen.

      I will go ahead to approve this project because it has exceeded the no of code review cycle

### This project is approved, go ahead and merge with master branch.

**Pull request merged**